client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,68 @@
+.scoreboard {
+  background: #fff;
+  color: #333;
+}
+
+.scoreboard > div.scoreboardIntermediate {
+  position: relative;
+  z-index: 0;
+  border: 2px solid black;
```

Comment:
Ideally this border would be moved up to the `.scoreboard` but unfortunately the inherited radius 2px would no longer be correct because the `.scoreboardIntermediate` element would then be 4px narrower so you can end up with slight gaps. So it has to be here to avoid introducing any gaps.